lib/lcdbwf/R/helpers.R

```diff
@@ -473,9 +479,14 @@ nested.lapply <- function(x, subfunc, ...){
 #'     ...
 #'   )
 #'
-#' `enrich_list` is optional. Note that `enrich_list`, if provided, must use
-#' results names available in `res_list`. In this example, the names are
-#' "ko.vs.wt" and "het.vs.wt". It has the following format:
+#' `enrich_list` is optional. Note that `enrich_list`, if provided, these are usually
+#' results names available in `res_list`. Alternatively, can have a 'res' key at the
+#' second-level containing a result name available in `res_list`.
```

Comment:
```suggestion
#' `enrich_list` is optional. If provided, has names corresponding to
#' results names in `res_list`. Alternatively, can have a 'res' key at the
#' second-level containing a result name available in `res_list`.
```

Is there a historical reason we need to support both ways of connecting to a results object?

Reply:
I don't have a preference for what should be default (or what would objectively be the "best"). Though I feel like a separate enrich_mapping would be the most flexible.

Reply:
Yes, agreed. A separate enrich_mapping would be nice, but will need some tweaking in carnation.
Will add that to a future update.